src/ui/public/field_format_editor/editors/url/url.html

```diff
@@ -7,6 +7,13 @@
   </select>
 </div>
 
+<div class="checkbox" ng-if="editor.formatParams.type == 'a'">
+  <label>
+    <input ng-model="editor.formatParams.openLinkInCurrentTab" class="ng-pristine ng-valid ng-touched" type="checkbox">
```

Comment:
There really isn't a reason to manually add these classes (`ng-pristine`, `ng-valid` and `ng-touched`). Angular will handle that automatically.